torch/ao/quantization/experimental/observer.py

```diff
@@ -133,10 +136,16 @@ def forward(self, x_orig):
         self.max_val = max_val
         return x_orig
 
-    def quant_levels_visualization(self, obs_result, filename):
+    r"""Displays visualization of APoT quantization levels
+        Args:
+            alpha: alpha APoT qparam (Tensor)
+            quantization_levels: quantization levels APoT qparam (Tensor)
+            level_indices: level indices APoT qparam (Tensor)
+    """
+    def quant_levels_visualization(self, alpha, quantization_levels, level_indices):
```

Comment:
are these arguments calculated from `calculate_qparams`? maybe we can simplify this method by calling `calculate_qparams` to get these arguments and then do the visualization?